dotcom-rendering/src/model/enhanceBlocks.ts

```diff
@@ -43,11 +45,18 @@ const enhanceNewsletterSignup =
 // example: enhanceInteractiveContentElements needs to be before enhanceNumberedLists
 // as they both effect SubheadingBlockElement
 export const enhanceElements =
-	(format: ArticleFormat, blockId: string, options: Options) =>
-	(elements: FEElement[]): FEElement[] =>
-		[
-			enhanceLists(enhanceElements(format, blockId, options)),
-			enhanceTimelines(enhanceElements(format, blockId, options)),
+	(
+		format: ArticleFormat,
+		blockId: string,
+		abTests: ServerSideTests,
+		options: Options,
```

Comment:
If at all possible, it would be preferable to keep the tests inside the `options` parameter.

While we do not have an explicitly codified rule around the number of parameters, methods are more readable if it is kept to a maximum of three, [as per the Deno style guide](https://docs.deno.com/runtime/manual/references/contributing/style_guide).

Reply:
Sounds good